### PR TITLE
[Inference][XPU] Add bn act fusion plugin

### PR DESCRIPTION
**Pull request by csy0225**
### PR types

Performance optimization
### PR changes

OPs
### Description

Add bn + act fusion pass and xpu plugin impl.

### CODE REVIEW

File: paddle/phi/infermeta/fusion.cc [hunk 1]
```diff
@@ -821,6 +821,102 @@ void FastLayernormXPUInferMeta(const MetaTensor& x,
   out->set_layout(x.layout());
 }
 
+void BnActXPUInferMeta(const MetaTensor& x,
```
Comment (hong19860320) [inline paddle/phi/infermeta/fusion.cc]: Bn -> BN ？

File: paddle/phi/infermeta/fusion.h [hunk 2]
```diff
@@ -201,6 +201,18 @@ void FastLayernormXPUInferMeta(const MetaTensor& x,
                                float epsilon,
                                MetaTensor* out);
 
+void BnActXPUInferMeta(const MetaTensor& x,
```
Comment (hong19860320) [inline paddle/phi/infermeta/fusion.h]: Bn->BN ?

**Review by hong19860320 — DISMISSED** [review]
LGTM

**Review by hong19860320 — APPROVED** [review]
LGTM

**Review by qili93 — APPROVED** [review]
LGTM for const_cast

**Review by XiaoguangHu01 — APPROVED** [review]
LGTM

**Review by hong19860320 — APPROVED** [review]
LGTM

**Pull request merged**